src/core/multiboot.h

```diff
@@ -40,6 +40,10 @@
 #define MULTIBOOT_ELF_SECTION_FLAG_ALLOCATED  0x2
 #define MULTIBOOT_ELF_SECTION_FLAG_EXECUTABLE 0x4
 
+#define MULTIBOOT_FRAMEBUFFER_TYPE_INDEXED  0
+#define MULTIBOOT_FRAMEBUFFER_TYPE_RGB      1
+#define MULTIBOOT_FRAMEBUFFER_TYPE_EGA_TEXT 2
```

Comment:
I see that we only support one mode. I think it would be best to focus on that mode and remove everything not related to it. That will make things easier to read

Reply:
Done